packages/server/graphql/mutations/helpers/generateGroups.ts

```diff
@@ -56,10 +57,19 @@ const generateGroups = async (
   }
 
   const r = await getRethink()
-  await r.table('NewMeeting').get(meetingId).update({autogroupReflectionGroups}).run()
+  const meetingRes = await r
+    .table('NewMeeting')
+    .get(meetingId)
+    .update(
+      {autogroupReflectionGroups},
+      {returnChanges: true}
+    )('changes')(0)('new_val')
+    .run()
+  const {facilitatorUserId} = meetingRes
   const data = {meetingId}
   const operationId = dataLoader.share()
   const subOptions = {operationId, mutatorId: SERVER_ID}
+  analytics.suggestGroupsGenerated(facilitatorUserId, meetingId, teamId)
```

Comment:
We require a userId in the `track` function, but in this case, there isn't really a userId. The meeting proceeds from the Reflect phase to the Group phase, and this function is triggered.

I've included the facilitator user id as I think that's the most accurate user id